cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -487,6 +487,8 @@ def retry_query(query_func: tp.Callable, timeout: int = 20) -> tp.Any:
             time.sleep(sleep_time)
         try:
             response = query_func()
+            if not response:
```

Comment:
not sure if there is a better way to handle it

Reply:
The query is supposed to raise the `AssertionError` when empty result is not expected. See `check_pool_off_chain_data`.